internal/backends/collection.go

```diff
@@ -224,6 +226,50 @@ func (cc *collectionContract) Stats(ctx context.Context, params *CollectionStats
 	return res, err
 }
 
+// ListIndexesParams represents the parameters of Collection.ListIndexes method.
+type ListIndexesParams struct{}
+
+// ListIndexesResult represents the results of Collection.ListIndexesResult method.
+type ListIndexesResult struct {
+	Indexes []IndexInfo
+}
+
+// IndexInfo represents information about a single index.
+type IndexInfo struct {
+	Name   string
+	Key    []IndexKeyPair
+	Unique bool
+}
+
+// IndexKeyPair consists of a field name and a sort order that are part of the index.
+type IndexKeyPair struct {
+	Field string
+	Order IndexOrder
```

Comment:
```suggestion
	Descending bool
```

To avoid any confusion